lib/Network/Linklater.hs

```diff
@@ -83,6 +87,16 @@ slash inner req respond = do
     Left msg ->
       respond (responseOf status400 ("linklater: unable to parse request: " <> msg ^. packed))
 
+trigger :: (Event -> Application) -> Application
```

Comment:
I think you meant to type `eventTrigger` here?

Reply:
Ughhh, that's what happens when I write last second stylistic changes without re-testing.